api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/AdditionalPregnancyPaymentMessageProcessor.java

```diff
@@ -59,6 +63,7 @@ public MessageStatus processMessage(Message message) {
         int paymentAmountInPence = calculatePaymentAmountInPence(message, context, paymentCycle);
         if (paymentAmountInPence > 0) {
             paymentService.makeInterimPayment(paymentCycle.get(), context.getClaim().getCardAccountId(), paymentAmountInPence);
+            reportPaymentMessageSender.sendReportTopUpPaymentMessage(paymentCycle.get().getClaim(), paymentCycle.get(), paymentAmountInPence);
```

Comment:
I think as this is going to be specific to pregnancy payments, the method should be renamed accordingly to be pregnancy specific. If its going to be more generic, then it should be moved into the makeInterimPayment method. Sounds like the former is correct.

Reply:
Renamed to sendReportPregnancyTopUpPaymentMessage().